webClient/src/app/shared/dialog/language-server/language-server.component.html

```diff
@@ -8,6 +8,8 @@
   
   Copyright Contributors to the Zowe Project.
 -->
+
+<button mat-dialog-close class="close-dialog-btn"><i class="fa fa-close"></i></button>
```

Comment:
Good on you for using the existing font awesome library that we have imported however!